cvss/cvss3.py

```diff
@@ -258,17 +263,34 @@ def compute_modified_isc_base(self):
                                       ),
                                      D('0.915'))
 
-    def compute_modified_isc(self):
+    def compute_modified_isc_30(self):
         """
+        This is CVSS:3.0 version
+
         If Modified Scope Unchanged    6.42 x [ISCModified]
-        If Modified Scope Changed    7.52 x [ISCModified-0.029] - 3.25 x [ISCModified-0.02]^15
+        If Modified Scope Changed      7.52 x [ISCModified-0.029] - 3.25 x [ISCModified-0.02]^15
```

Comment:
It is a little embarrassing but when I saw this comment last week I did not immediately understand that it was the formula being used.  Also it may be better as a code comment than as part of the docstring.

Reply:
You are right. However, it is everywhere in the code and for some reason I thought it is a good idea back when I wrote the first version.